### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by vinkla**
1. Removed default homepage url (packagist will use the github url by default)
2. Update the php version requirement

### CODE REVIEW

**Comment by denis-sokolov** [comment]
Is the intent with the version change to not mistakenly claim we support PHP 8?

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "license": "MIT",
     "description": "php error handling for cool kids",
     "keywords": ["library", "error", "handling", "exception", "whoops", "zf2"],
-    "homepage": "https://github.com/filp/whoops",
```
Comment (staabm) [inline composer.json]: we should instead use https://filp.github.io/whoops/ as the url

Comment (vinkla) [inline composer.json]: Oh, yeah. True! I'll fix this.

**Comment by vinkla** [comment]
@denis-sokolov yeah, we don't know if the library will work on PHP 8.0 once it is released. This way we can control that.

**Pull request merged**